net/http/impl/server.h

```diff
@@ -335,14 +336,23 @@ class HTTPServerConnection {
     connection_.BlockingWrite(kCRLF);
```

Comment:
This `kCRLF` is wrong here. It's not included into the measured `Content-Length` and it's not in the specs either: http://www.w3.org/Protocols/rfc2616/rfc2616-sec6.html